### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by LucsT**
Pr to test packaging, build and co.

### CODE REVIEW

File: manifest.konnector [hunk 1]
```diff
@@ -49,8 +48,8 @@
   "langs": ["fr", "en"],
   "locales": {
     "fr": {
-      "short_description": "Template de connecteur",
-      "long_description": "Ce template récupère une liste de livres sur un site de scraping",
+      "short_description": "connecteur Just-Eat",
+      "long_description": "Ce template récupère vos facture sur le site de https://www.just-eat.fr/",
```
Comment (enguerran) [inline manifest.konnector]: s/Ce template/Ce connecteur/

File: manifest.konnector [hunk 2]
```diff
@@ -67,8 +66,8 @@
       }
     },
     "en": {
-      "short_description": "Connector template",
-      "long_description": "This template fetches a list of books from a scraping website",
+      "short_description": "Just-Eat Connector",
+      "long_description": "This template fetches your bills from https://www.just-eat.fr/",
```
Comment (enguerran) [inline manifest.konnector]: s/This template/This connector/

File: manifest.konnector (hunk 1 above)
Comment (LucsT) [reply manifest.konnector]: Bien vu, merci

File: .travis.yml [hunk 3]
```diff
@@ -31,9 +35,8 @@ deploy:
...
     tags: true
 before_install:
-# run the yarn travisDeployKey command to genrate this
-# - openssl aes-256-cbc -K $encrypted_0b21ecdcf920_key -iv $encrypted_0b21ecdcf920_iv
-#   -in github_deploy_key.enc -out /tmp/github_deploy_key -d
+- openssl aes-256-cbc -K $encrypted_a8e1e885122a_key -iv $encrypted_a8e1e885122a_iv
+  -in github_deploy_key.enc -out /tmp/github_deploy_key -d
 - eval "$(ssh-agent -s)"
 - chmod 600 /tmp/github_deploy_key
 - ssh-add /tmp/github_deploy_key
```
Comment (doubleface) [inline .travis.yml]: The part:

```yml
after_deploy:
- rm /tmp/github_deploy_key
- ssh-add -D
```

From the template is missing

**Pull request merged**